example/models.py

```diff
@@ -8,12 +12,34 @@ def auto_alts():
     return asim.identity_matrix(["cars%d" % i for i in range(5)])
 
 
+@sim.table()
+def zones():
+    # I grant this is a weird idiom but it helps to name the index
+    return pd.DataFrame({"TAZ": np.arange(1454)+1}).set_index("TAZ")
+
+
+@sim.injectable()
+def nonmotskm_omx():
+    return omx.open_omxfile('data/nonmotskm.omx')
```

Comment:
This will be `omx.openFile` once you're using [openmatrix](https://pypi.python.org/pypi/OpenMatrix).